src/neo/SmartContract/Native/NativeContract.cs

```diff
@@ -14,32 +14,36 @@ namespace Neo.SmartContract.Native
     public abstract class NativeContract
     {
         private static readonly List<NativeContract> contractsList = new List<NativeContract>();
-        private static readonly Dictionary<string, NativeContract> contractsNameDictionary = new Dictionary<string, NativeContract>();
```

Comment:
Maybe we should have both dictionaries, we will need to convert the name to id for RPC

Reply:
The name is not unique. You can't convert it to id.

Reply:
But it will be unique for native contracts, with RPC it will be easier to use 'neo' instead of a number.

Reply:
For native contracts, you can easily mapping the name to the contracts.